ext/options.html

```diff
@@ -1,43 +1,63 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<script src="javascripts/TabCarousel.js"></script>
-<style>
-body {
-  font-family: "Trebuchet MS", sans-serif;
-}
+    <script src="javascripts/TabCarousel.js"></script>
+    <style>
+        body {
+            font-family: "Trebuchet MS", sans-serif;
+        }
 
-th {
-  font-weight: normal;
-  text-align: right;
-}
-</style>
+        th {
+            font-weight: normal;
+            text-align: right;
+        }
+    </style>
 </head>
+
 <body>
-<h1>TabCarousel Options</h1>
+    <h1>TabCarousel Options</h1>
 
-<form id="options">
-<table>
-<tr>
-<th><label for="flipWait_ms">Flip Wait (ms):</label></th>
-<td><input atuofocus id="flipWait_ms" name="flipWait_ms" type="number" min="1000" step="1000" /></td>
-</tr>
-<tr>
-<th><label for="automaticStart">Start automatically:</label></th>
-<td><input id="automaticStart" name="automaticStart" type="checkbox" /></td>
-</tr>
-</table>
-<br />
-<input type="submit" value="Save" />
-<span id="status"></span>
-</form>
+    <form id="options">
+        <table>
+            <tr>
+                <th><label for="flipWait_ms">Flip Wait (ms):</label></th>
+                <td><input autofocus
+                           id="flipWait_ms"
+                           name="flipWait_ms"
+                           type="number"
+                           min="1000"
+                           step="1000" /></td>
+            </tr>
+            <tr>
+                <th><label for="automaticStart">Start automatically:</label></th>
+                <td><input id="automaticStart"
+                           name="automaticStart"
+                           type="checkbox" /></td>
+            </tr>
+            <table>
+                <tr>
+                    <th><label for="disableReload">Disable Automatic Reload:</label></th>
+                    <td><input id="disableReload"
+                               name="disableReload"
+                               type="checkbox" /></td>
+                </tr>
+            </table>
+            <br />
+            <input type="submit"
+                   value="Save" />
+            <span id="status"></span>
+    </form>
 
-<p>
-<a href="https://github.com/benjaminoakes/TabCarousel/wiki">Wiki</a> | <a href="https://github.com/benjaminoakes/TabCarousel/issues">Report a bug or request a feature</a> | <a href="https://github.com/benjaminoakes/TabCarousel">Source Code</a>
-</p>
+    <p>
+        <a href="https://github.com/benjaminoakes/TabCarousel/wiki">Wiki</a> | <a
+           href="https://github.com/benjaminoakes/TabCarousel/issues">Report a bug or request a feature</a> | <a
+           href="https://github.com/benjaminoakes/TabCarousel">Source Code</a>
+    </p>
 
-<script>
-new TabCarousel.OptionsController(document.forms['options']);
-</script>
+    <script>
+        new TabCarousel.OptionsController(document.forms['options']);
+    </script>
 </body>
-</html>
+
+</html>
```

Comment:
I might be interested in these formatting changes, but for now, could you limit the changes to the ones necessary for the feature, please?